<img src="https://userweb.fct.unl.pt/~jmc.xavier/MAI/iLogos/logo_novafct.png" width="200">

# Departamento de Engenharia Mecânica e Industrial
## Mecânica Aplicada I 

## CENTROIDES E CENTROS DE GRAVIDADE

### Problema 7 <a name="P7"></a>

Considere a superfície $S$ da figura delimitada por o eixo $y$, um ramo de parábola e uma reta inclinada a 45$^\circ$. Responda às seguintes alíneas:

[a)](#A) Determine para o referencial indicado na figura as coordenadas $x$ e $y$ do centroíde da superfície $S$ utilizando o método de integração.

[b)](#B) Determine o volume do sólido de revolução obtido pela rotação da superfície indicada na figura em torno do eixo $x$. Enuncie os teoremas de Pappus-Guldinus.

[c)](#C) Sem efetuar cálculos (mas apresentando a formulação) explique como calcularia a área da superfície do sólido de revolução gerado pela rotação em torno de $y$.

<img src="https://userweb.fct.unl.pt/~jmc.xavier//MAI/Notebooks/Ch05/P7/MAI_Ch05_P7.jpg"
width="500">

### Resolução

#### (a) Determine para o referencial indicado na figura as coordenadas $x$ e $y$ do centroíde da superfície $S$ utilizando o método de integração. >([_enunciatum_](#P7))

Para recorrer ao método da integração é necessário conhecer todos os limites de integração. Por conseguinte, antes de realizar qualquer integração, determina-se as coordenadas do ponto relativo à interceção entre a parábola e a reta.

<img src="https://userweb.fct.unl.pt/~jmc.xavier/MAI/Notebooks/Ch05/P7/MAI_Ch05_P7a.jpg"
style="max-width: 100%;"/>

Este ponto é determinado pela solução da seguinte expressão:

\begin{equation}
\begin{aligned}
f(x) = 9-x^2~, \quad g(x) = x \quad : \quad
f(x) = g(x)~~ \therefore ~~ x_1 =2.541
\end{aligned}
\end{equation}

In [1]:
import numpy as np
import sympy as sy
from sympy.solvers import solve

x, y = sy.symbols('x y')

f = 9 - x**2
g = x
sol = solve(f-g,x)
x1 = sol[0]
print('x1 = ',x1,' = ','{:.3f}'.format(sy.N(x1)))
x2 = sol[1]
print('x2 = ',x2,' = ','{:.3f}'.format(sy.N(x2)))

x1 =  -1/2 + sqrt(37)/2  =  2.541
x2 =  -sqrt(37)/2 - 1/2  =  -3.541


A área da superfície confinada pela parábola e pela reta pode ser determinada resolvendo a seguinte formula integral:

<img src="https://userweb.fct.unl.pt/~jmc.xavier/MAI/Notebooks/Ch05/P7/MAI_Ch05_P7b.jpg"
style="max-width: 100%;"/>

\begin{equation}
\begin{aligned}
A &= \int\mathrm{d}A =
\int_{0}^{x_1}\int_{x}^{9-x^2}
\mathrm{d}y\mathrm{d}x =
\int_{0}^{x_1}\left[y \right]_{x}^{9-x^2}
\mathrm{d}x \\
& = \int_{0}^{x_1}\left(9-x^2-x \right)
\mathrm{d}x =
\left[9x - \frac{x^3}{3} - \frac{x^2}{2} \right]_{0}^{x_1} \\
&= 9(2.54) - \frac{2.54^3}{3} - \frac{2.54^2}{2}
= 14.17~\textrm{m}^3
\end{aligned}
\end{equation}

In [2]:
li = g
lf = f
fa = 1
A = sy.integrate(fa, (y,li,lf), (x,0,x1))
print('A =',sy.factor(A),' m2')
print('A =','{:.3f}'.format(sy.N(A)),' m2')

A = (-55 + 37*sqrt(37))/12  m2
A = 14.172  m2


O momento estático (ou de primeira ordem de superfície) em relação ao eixo $x$ vem:

\begin{equation}
\begin{aligned}
Q_x &= \int y\,\mathrm{d}A =
\int_{0}^{x_1}\int_{x}^{9-x^2}y\,
\mathrm{d}y\mathrm{d}x =
\int_{0}^{x_1}\left[\frac{y^2}{2} \right]_{x}^{9-x^2}
\mathrm{d}x \\
& = \int_{0}^{x_1}\left[\frac{(9-x^2)^2}{2} - \frac{x^2}{2} \right]
\mathrm{d}x = \frac{1}{2}\int_{0}^{x_1}\left[81 - 19x^2 + x^4
\right]
\mathrm{d}x \\
&= \frac{1}{2}\left[81 x-\frac{19}{3}x^3
+ \frac{x^5}{5}
\right]_{0}^{x_1}
= 61.55~\textrm{m}^3
\end{aligned}
\end{equation}

In [3]:
fa = y
Qx = sy.integrate(fa, (y,li,lf), (x,0,x1))
print('Qx =',sy.factor(Qx),' m2')
print('Qx =','{:.3f}'.format(sy.N(Qx)),' m2')

Qx = (23 + 148*sqrt(37))/15  m2
Qx = 61.550  m2


Pelo que a ordenada do centroide da superfície vem:

\begin{equation}
\overline{Y}
= \frac{Q_x}{A}
= \frac{\displaystyle\int y\,\mathrm{d}A}
{\displaystyle\int \mathrm{d}A}
= 4.34~\textrm{m}
\end{equation}

In [4]:
ycent = Qx/A
print('ycent =',sy.factor(ycent),' m')
print('ycent =','{:.3f}'.format(sy.N(ycent)),' m')

ycent = 4*(23 + 148*sqrt(37))/(5*(-55 + 37*sqrt(37)))  m
ycent = 4.343  m


Por sua vez, o  momento estático em relação ao eixo $y$ vem:

\begin{equation}
\begin{aligned}
Q_y &= \int x\,\mathrm{d}A =
\int_{0}^{x_1}\int_{x}^{9-x^2}x\,
\mathrm{d}y\mathrm{d}x =
\int_{0}^{x_1}x\left[y\right]_{x}^{9-x^2}
\mathrm{d}x \\
& = \int_{0}^{x_1}x\left(9-x^2-x\right)
\mathrm{d}x = \int_{0}^{x_1}\left(9x-x^3-x^2
\right)
\mathrm{d}x \\
&= \left[\frac{9x^2}{2} -\frac{x^4}{4} - \frac{x^3}{3}
\right]_{0}^{x_1}
= 13.16~\textrm{m}^3
\end{aligned}
\end{equation}

In [5]:
fa = x
Qy = sy.integrate(fa, (y,li,lf), (x,0,x1))
print('Qy =',sy.factor(Qy),' m2')
print('Qy =','{:.3f}'.format(sy.N(Qy)),' m2')

Qy = -(-55 + sqrt(37))*(-1 + sqrt(37))**2/96  m2
Qy = 13.164  m2


Pelo que a ordenada do centroide da superfície vem:

\begin{equation}
\overline{X} = \frac{Q_y}{A}
= \frac{\displaystyle\int x\,\mathrm{d}A}
{\displaystyle\int \mathrm{d}A}
= 0.93~\textrm{m}
\end{equation}

In [6]:
xcent = Qy/A
print('xcent =',sy.factor(xcent),' m')
print('xcent =','{:.3f}'.format(sy.N(xcent)),' m')

xcent = -(-55 + sqrt(37))*(-1 + sqrt(37))**2/(8*(-55 + 37*sqrt(37)))  m
xcent = 0.929  m


#### (b) Determine o volume do sólido de revolução obtido pela rotação da superfície indicada na figura em torno do eixo $x$. Enuncie os teoremas de Pappus-Guldinus. ([_enunciatum_](#P7))

**Teorema I**

A área de uma superfície de revolução é igual ao produto do comprimento da curva geratriz pelo caminho percorrido pelo centroide da curva durante o movimento de rotação que gera a superfície:

\begin{equation}
A = \int \mathrm{d}A
= \int 2\pi\,y\,\mathrm{d}L = 2\pi\,\overline{Y}\,L
\qquad\vee\qquad
A = 2\pi\,\overline{X}\,L
\end{equation}

**Teorema II**

O volume de um corpo de revolução é igual ao produto da área da superfície geratriz pelo caminho percorrido pelo centroide dessa superfície durante o movimento de rotação que gera o corpo:

\begin{equation}
V = \int \mathrm{d}V
= \int 2\pi\,y\,\mathrm{d}A
= 2\pi\,\overline{Y}\,A
\qquad\vee\qquad
V = 2\pi\,\overline{X}\,A
\end{equation}

De acordo com o Teorema II de Pappus-Guldinus o volume do sólido de revolução obtido pela rotação da superfície em torno do eixo $x$ resulta:

\begin{equation}
V = 2\pi\,\overline{Y}A = 2\pi\,(4.34)\,(14.17) = 386.73~\textrm{m}^3
\end{equation}

In [7]:
V = 2*np.pi*ycent*A
print('V =',sy.factor(V),' m3')
print('V =','{:.3f}'.format(sy.N(V)),' m3')

V = 61.9940950308386*(0.155405405405405 + 1.0*sqrt(37))  m3
V = 386.730  m3


#### (c) Sem efetuar cálculos (mas apresentando a formulação) explique como calcularia a área da superfície do sólido de revolução gerado pela rotação em torno de $y$.  ([_enunciatum_](#P7))

Recorrendo ao Teorema I de Pappus-Guldin, a área da superfície do sólido de revolução gerado pela rotação em torno de $y$ pode ser calculada de acordo com:

\begin{equation}
A = 2\pi\overline{X}L~~\textrm{(m$^2$)}
= 2\pi \sum \overline{x}_i\cdot L_i
\qquad\wedge\qquad
\sum \overline{x}_i\cdot L_i
= x_\mathrm{AB} L_\mathrm{AB} + x_\mathrm{BC}L_\mathrm{BC}
\end{equation}

onde $L_\mathrm{AB}$ e $L_\mathrm{BC}$ são os comprimentos dos troços AB e BC delimitados pela parábola e reta, respectivamente; e $x_\mathrm{AB}$ e $x_\mathrm{BC}$ são as distâncias entre os centróides ($G_\mathrm{AB}$ e $G_\mathrm{BC}$) e o eixo $y$ dos troços AB e BC, respectivamente.

A área da superfície do sólido de revolução gerado pela rotação em torno de $y$ vem portanto:

\begin{equation}
A = 2\pi\overline{X}L
= 2\pi (x_\mathrm{AB} L_\mathrm{AB} +
x_\mathrm{BC}L_\mathrm{BC})
\end{equation}

<img src="https://userweb.fct.unl.pt/~jmc.xavier/MAI/Notebooks/Ch05/P7/MAI_Ch05_P7c.jpg"
style="max-width: 100%;"/>

#### Anexo: Formulário

**Centroide de áreas comuns**

<img src="https://userweb.fct.unl.pt/~jmc.xavier//MAI/Notebooks/GFormulas/Centroids_areas.jpg"
width="900">

<img src="https://userweb.fct.unl.pt/~jmc.xavier//MAI/Notebooks/GFormulas/CentroidTables01.gif"
width="900">

---

Copyright (c) DEMI - FCT NOVA

Interactive computing by <a href="https://jupyter.org/" target="_blank"> <span
style="color:#333399"> Jupyter Notebook </span> </a> &nbsp;|&nbsp;Coded by <a href = "mailto: jmc.xavier@fct.unl.pt">José Xavier</a>

Licensed under  <a href="http://creativecommons.org/licenses/by-sa/4.0/"
target="_blank"> <span style="color:#333399;font-size: 20px"> CC BY-SA 4.0  </span></a>